AppCreationScripts/apps.json

```diff
@@ -48,7 +48,9 @@
 		      "replaceTokens": {
              "appId": "your_client_id",
              "tenantId": "Enter_the_Tenant_Name_Here",
-             "clientSecret": "The secret generated by AAD during your confidential app registration"
+             "clientSecret": "The secret generated by AAD during your confidential app registration",
```

Comment:
Can we make this a shorter value? (similar to your_client_id).

Reply:
Sure, this one was written by @jmprieur. Unless he has any specific reason behind it, we can change it. This will mean changing parameters.json and App Creation scipts too. For now we can change parameters.json and apps.json and create an issue to update App Creation Scripts

Reply:
@navyasric @abhidnya13 If you decide to change the value in "key":"value" pair, ensure that the corresponding file where the actual replacement will happen is consistent with value specified in apps.json. Otherwise, the Azure portal will not be able to find the string to replace.

Reply:
@abhidnya13 Let's address this in another PR. For now, this should be ok as is.

Reply:
Sounds good :)